script.js

```diff
@@ -170,14 +171,47 @@ function closePopup() {
   const unpopElement = document.getElementById('desk-popup');
   unpopElement.parentNode.removeChild(unpopElement);
 }
+
+// Local Storage
 const contactForm = document.getElementById('contact-form-validate');
 contactForm.addEventListener('submit', (event) => {
   event.preventDefault();
   document.getElementById('error-msg').innerText = '';
   const email = document.getElementById('form-email').value;
+  const formData = {
+    name: document.getElementById("fullname").value,
+    email: email,
+    message: document.getElementById("text-area").value,
+  }
+  localStorage.setItem("key",JSON.stringify(formData))
+  
   if (email !== email.toLowerCase()) {
     document.getElementById('error-msg').innerText = 'Email Address should be in Lower case';
   } else {
     document.forms[0].submit();
+    localStorage.clear();
   }
 });
+
+// const email = document.getElementById("form-email");
+//  email.addEventListener('input', (event) => {
+//     const formemail = email.value;
+//     localStorage.setItem("emailkey", formemail);
+// });
+
```

Comment:
- [ ] Kindly remove all commented out blocks of code to keep your code clean and readable